browser/brave_tab_helpers.cc

```diff
@@ -45,10 +47,12 @@ void AttachTabHelpers(content::WebContents* web_contents) {
 #if defined(OS_ANDROID)
   DesktopModeTabHelper::CreateForWebContents(web_contents);
   BackgroundVideoPlaybackTabHelper::CreateForWebContents(web_contents);
-#else
+#endif
+
 #if BUILDFLAG(BRAVE_REWARDS_ENABLED)
   brave_rewards::RewardsTabHelper::CreateForWebContents(web_contents);
 #endif
+#if !defined(OS_ANDROID)
```

Comment:
can we move these up to keep the if/else? It's harder to read this way

Reply:
Yep, it was messed up after rebase.